marconi-chain-index/test-compare-cardano-db-sync/Spec.hs

```diff
@@ -14,9 +9,9 @@
 
      - CARDANO_NODE_SOCKET_PATH
      - CARDANO_NODE_CONFIG_PATH
+     - DBSYNC_PG_URL
      - MARCONI_DB_DIRECTORY_PATH
-     - DBSYNC_PGPASSWORD: The default password for cardano-db-sync's postgres database is in its repo in the file: config/secrets/postgres_password
```

Comment:
Why isn't it required anymore?

Reply:
Sorry for the long delay.
I am using URL, connection string for PG stuff, It is One environment variable rather than userid and password.